modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -290,10 +293,10 @@ public Response getBatchAssignments(@PathParam("applicationName")
                     headers, experimentBatch);
         }
 
-        List<Map> myAssignments = assignments.doBatchAssignments(userID, applicationName, context, createAssignment, FALSE,
-                headers, experimentBatch, null, null);
+        List<Assignment> myAssignments = assignments.doBatchAssignments(userID, applicationName, context, createAssignment, FALSE,
+                headers, experimentBatch);
 
-        return httpHeader.headers().entity(ImmutableMap.<String, Object>builder().put("assignments", myAssignments).build()).build();
+        return httpHeader.headers().entity(ImmutableMap.<String, Object>builder().put("assignments", toBatchAssignmentResponseMap(myAssignments)).build()).build();
```

Comment:
Cosmetic: Can we move all the string literals like `"assignments"` in this case to a string constant defined at the beginning of the class. This makes it less error prone and easy to reference and change.